### ISSUE TITLE
Do not build WHERE strings manually (code quality)

### ISSUE THREAD

**Issue by JeroenDeDauw** [inblockio/DataAccounting#152]
Notice the line before `__METHOD__`

Bad:

```php
$row = $dbr->selectRow(
	'page_verification',
	[ 'rev_id', 'verification_hash', 'signature', 'public_key', 'wallet_address', 'witness_event_id' ],
	"rev_id = $revId",
	__METHOD__
);
```

Good:

```php
$row = $dbr->selectRow(
	'page_verification',
	[ 'rev_id', 'verification_hash', 'signature', 'public_key', 'wallet_address', 'witness_event_id' ],
	[ 'rev_id' => $revId ],
	__METHOD__
);
```

First one allows for SQL injection if the user gets control over $revId. Second one does escaping.

**Comment by rht** [comment]
Yeah, that was the old pattern that still exists in some of the DB operation. I have tried to eliminate them when I spot them.

**Comment by rht** [comment]
I think you have spotted and removed the last one in #153. I did `rg -tphp "'.*'.*=.*," | rg -v '>'` (and also for the single quote version). This issue can be closed when #153 is merged.
